android/titanium/src/java/org/appcelerator/titanium/view/TiUIView.java

```diff
@@ -1190,23 +1183,51 @@ protected boolean canApplyTouchFeedback(@NonNull KrollDict props)
 
 	/**
 	 * Applies touch feedback. Should check canApplyTouchFeedback() before calling this.
-	 * @param backgroundColor The background color of the view.
-	 * @param rippleColor The ripple color.
+	 * @param rippleColor The ripple color to use. Set to null to use system's default ripple color.
 	 */
-	private void applyTouchFeedback(@NonNull Integer backgroundColor, @Nullable Integer rippleColor)
+	private void applyTouchFeedback(Integer rippleColor)
```

Comment:
Any particular reason to drop the `@Nullable`? I kind of like having it be explicit in our APIs so we can use tools to verify checking for null pointers or not...

Reply:
Personally, I question the value of the `@Nullable` annotation. All reference types are implicitly nullable. The only benefit it provides is more for IDE intellisense purposes since it reveals the annotations and effectively documents the API which we can already do via JavaDoc comments.

The `@NonNull` annotation is far more valuable since we can use it to trigger compiler/linter errors. So, if it doesn't have the `@NonNull` annotation, then it's assumed nullable.

Anyways, that's my 2 cents. Unless you can think of something else that might benefit from this?

Reply:
I get that things are implicitly Nullable (so Lon as they're not primitives). I saw it more as a marker of what APIs we've "done" since we've got this large legacy codebase that we didn't use the annotations on. If we don't use `@Nullable`, it's unclear if we've already reviewed the API and decided they're all implicitly nullable, or if we simply haven't gotten around to adding annotations.